relevanceai/api/batch/batch_insert.py

```diff
@@ -30,6 +25,8 @@
 from relevanceai.package_utils.logger import FileLogger
 from relevanceai.package_utils.progress_bar import progress_bar
 from relevanceai.package_utils.utils import Utils
+from relevanceai.dataset.crud.helpers import make_id

```

Comment:
An argument can be made here I think for `package_utils.make_id`

Reply:
yah you're probs right